doc/dev/cephfs-mirroring.rst

```diff
@@ -441,53 +455,66 @@ E.g., adding a regular file for synchronization would result in failed status::
+
+When mirroring is disabled, the ``fs mirror status`` command for the file
+system will not show up in command help.
 
-When mirroring is disabled, the respective `fs mirror status` command for the file system
-will not show up in command help.
+The mirroring module provides a couple of commands to display directory mapping
+and distribution information. To check which mirror daemon a directory has been
+mapped to use, run a command of the following form:
 
-Mirroring module provides a couple of commands to display directory mapping and distribution
-information. To check which mirror daemon a directory has been mapped to use::
+.. prompt:: bash $
+
+   ceph fs snapshot mirror dirmap cephfs /d0/d1/d2
+::
 
-  $ ceph fs snapshot mirror dirmap cephfs /d0/d1/d2
   {
     "instance_id": "404148",
     "last_shuffled": 1601284516.10986,
     "state": "mapped"
   }
 
-.. note:: `instance_id` is the RADOS instance-id associated with a mirror daemon.
+.. note:: ``instance_id`` is the RADOS instance-id associated with a mirror
+   daemon.
 
-Other information such as `state` and `last_shuffled` are interesting when running
-multiple mirror daemons.
+Other information such as ``state`` and ``last_shuffled`` are interesting when
+running multiple mirror daemons.
 
-When no mirror daemons are running the above command shows::
+If no mirror daemons are running, the same command shows the following:
+
+.. prompt:: bash $
+
+   ceph fs snapshot mirror dirmap cephfs /d0/d1/d2
+::
 
-  $ ceph fs snapshot mirror dirmap cephfs /d0/d1/d2
   {
     "reason": "no mirror daemons running",
     "state": "stalled"
   }
 
-Signifying that no mirror daemons are running and mirroring is stalled.
+This signifies that no mirror daemons are running and that mirroring is
+stalled.
 
 Re-adding Peers
 ---------------
 
-When re-adding (reassigning) a peer to a file system in another cluster, ensure that
-all mirror daemons have stopped synchronization to the peer. This can be checked
-via `fs mirror status` admin socket command (the `Peer UUID` should not show up
-in the command output). Also, it is recommended to purge synchronized directories
-from the peer  before re-adding it to another file system (especially those directories
-which might exist in the new primary file system). This is not required if re-adding
-a peer to the same primary file system it was earlier synchronized from.
+When re-adding (reassigning) a peer to a file system in another cluster, ensure
+that all mirror daemons have stopped synchronizing with the peer. This can be
+checked via the  ``fs mirror status`` admin socket command (the ``Peer UUID``
+should not show up in the command output). We recommend purging 
+synchronized directories from the peer before re-adding them to another file
+system (especially those directories which might exist in the new primary file
+system). This is not required if you re-add a peer to the same primary file
+system it was synchronized from before.
```

Comment:
from which?

Reply:
I agree with just about all of these, and will cram them into future PRs during tedious coming days.